capabilities/project-management.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: capability
+
+asset-name: project_management
+
+description: "Offload project management to us and we’ll watch over the day-to-day execution  so you can focus on bigger stuff."
```

Comment:
Remove a space after:
Offload project management to us and we’ll watch over the day-to-day execution

Reply:
Yes

Reply:
Fixed.